app/last-dose/last-dose.scss

```diff
@@ -0,0 +1,3 @@
+.last-dose {
+  color: color($colors, success)
```

Comment:
This is how you use a color from the color scheme defined in `app.variables.scss`